tc-health-client/README.md

```diff
@@ -176,6 +177,12 @@ Sample configuarion file:
   will be marked down when the number of consecutive polls reaches this threshold
   with the parent reported as unhealthy.  The default threshold is 2.
 
+### markup-poll-threshold
+
+  This controls when an healthy parent is marked up.  An healthy parent
```

Comment:
Typo "An"

Reply:
FWIW that also has some extra spaces at the beginning of the line. It doesn't affect rendering, they just get stripped so they're totally superfluous.

Reply:
@ocket8888 all the non mark lines with leading spaces?  Are you asking that I remove them?

Reply:
@ocket8888 I removed the leading spaces.  @jpappa200 I fixed the typo

Reply:
Oh, it would probably be most consistent to remove them everywhere then, but to be clear I'm not asking for anything, it's just something I noticed when I looked at the notification I got for an update on this PR.